src/main/java/com/gamegoo/controller/board/BoardController.java

```diff
@@ -111,4 +108,14 @@ public ApiResponse<String> delete(@PathVariable Long postId
 
         return ApiResponse.onSuccess("게시글을 삭제하였습니다.");
     }
+
+    @GetMapping("")
+    @Operation(summary = "게시판 글 목록 조회 API", description = "게시판에서 글 목록을 조회하는 API 입니다.")
+    @Parameter(name = "pageIdx", description = "조회할 페이지 번호를 입력해주세요.")
+    public ApiResponse<List<BoardResponse.boardListResponseDTO>> boardList(@RequestParam(defaultValue = "1") int pageIdx){
+        int pageSize = 20;
```

Comment:
페이지 사이즈는 어차피 요청마다 달라지는게 아니라 항상 고정값인데, 그럼 아예 BoardService 안에서
```suggestion
            private final static int PAGE_SIZE = 20;
```
이렇게 정의해서 사용하는 건 어떨까요?